packages/sdks/overrides/vue/src/functions/get-block-actions.ts

```diff
@@ -1,26 +0,0 @@
-import { evaluate } from './evaluate';
-
-export function getBlockActions(options: {
-  block: any;
-  context: any;
-  state: any;
-}): any {
-  const obj: any = {};
-  if (options.block.actions) {
-    for (const key in options.block.actions) {
-      if (!options.block.actions.hasOwnProperty(key)) {
-        continue;
-      }
-
-      const value = options.block.actions[key];
-      obj['v-on:' + key.toLowerCase()] = (event: Event) =>
```

Comment:
This file as duplicated just for the sake of this key value, so I added a separate fn `getEventHandlerName` and overwrote just that instead.

Reply:
awesome

Reply:
great idea